packages/ng/forms/switchfield/switchfield.component.ts

```diff
@@ -0,0 +1,40 @@
+import { booleanAttribute, Component, Input } from '@angular/core';
+import { FormFieldComponent } from '../../form-field/form-field.component';
+import { AbstractFieldComponent } from '../abstract-field-component';
+import { NoopValueAccessorDirective } from '../noop-value-accessor.directive';
+import { ReactiveFormsModule, Validators } from '@angular/forms';
+import { SafeHtml } from '@angular/platform-browser';
+import { InlineMessageState } from '../../inline-message/inline-message-state';
+import { FormFieldSize } from '../../form-field/form-field-size';
+import { InputDirective } from '../../form-field/input.directive';
+
+@Component({
+	selector: 'lu-switchfield',
+	standalone: true,
+	imports: [FormFieldComponent, ReactiveFormsModule, InputDirective],
+	templateUrl: './switchfield.component.html',
+	hostDirectives: [NoopValueAccessorDirective],
+})
+export class SwitchfieldComponent extends AbstractFieldComponent {
+	@Input({ required: true })
+	label: string;
+
+	@Input({ transform: booleanAttribute })
```

Comment:
TIL ! Is this necessary even the property is a boolean ?

Reply:
This allows you to do things like `<lu-switchfield hiddenLabel="false">...` without having compiler error because type string is not assignable to type boolean. Can also allow syntaxes like `<lu-switchfield hiddenLabel>...`.

Reply:
It's not 😄